src/hand-tracking/src/DrawParticlesImportanceThreshold.cpp

```diff
@@ -72,7 +72,7 @@ void DrawParticlesImportanceThreshold::predictStep(const ParticleSet& prev_parti
         else
             tmp_state = prev_particles.state(j);
 
-        if (!getSkipState() && prev_particles.weight(j) <= threshold)
+        if (!getSkipState() && exp(prev_particles.weight(j)) <= threshold)
```

Comment:
what `exp` function is this? `std::exp`?

Reply:
Yes, it is the `std` one. I will enforce it!